components/compliance-service/inspec/cli.go

```diff
@@ -67,6 +67,11 @@ func Scan(paths []string, target *TargetConfig, timeout time.Duration, env map[s
 	env["HOME"] = os.Getenv("HOME")
 
 	args := append([]string{binName, "exec"}, paths...)
+
+	for k, v := range inputs {
+		args = append(args, fmt.Sprintf("--input %s=%s", k, v))
```

Comment:
If more than one input is passed, only the last one will be used by inspec when this format is used:

```
inspe exec myinputs-test --input inputname1=inputvalue1 --input inputname2=inputvalue2
```

Based on the inspec help, I tested and this works with multiple inputs:
```
inspec exec myinputs-test --input "bucket_name"="aaa" "bucket_location"="ea aa"
```
I added quotes to avoid having trouble due to special bash characters, spaces chars, dollar sign, semicolon, etc.

Probably best to pass an inputs file to avoid shell injection. Let me study that a bit.

Reply:
This works and feels less risky due to special bash characters that can go into these inputs:

```
$ cat /tmp/inputs.yaml
bucket_name: nameee
bucket_location: locccc

$ inspec exec myinputs-test --input-file /tmp/inputs.yaml
```